bin/clair-scan

```diff
@@ -1,43 +1,26 @@
 #!/usr/bin/env bash
+set -xe
 
-# Make sure you installed and run
-# docker run -p 5432:5432 -d --name db arminc/clair-db:latest
-# and
-# docker run -p 6060:6060 --link db:postgres -d --name clair arminc/clair-local-scan:latest
+# Install clair-scanner
 
-RELEASE_TAG=0.10.0-rc.1
-declare -a TARGET_IMAGES=("ap-base"
-                          "ap-airflow"
-                          "ap-alertmanager"
-                          "ap-cadvisor"
-                          "ap-curator"
-                          "ap-elasticsearch"
-                          "ap-elasticsearch-exporter"
-                          "ap-fluentd"
-                          "ap-grafana"
-                          "ap-kibana"
-                          "ap-kube-replicator"
-                          "ap-kube-state"
-                          "ap-nginx"
-                          "ap-nginx-es"
-                          "ap-pgbouncer"
-                          "ap-pgbouncer-exporter"
-                          "ap-prisma"
-                          "ap-prometheus"
-                          "ap-redis"
-                          "ap-registry"
-                          "ap-statsd-exporter")
+if [[ ! -f $(which clair-scanner) ]]; then
+    wget https://github.com/arminc/clair-scanner/releases/download/v12/clair-scanner_linux_amd64 -O /usr/local/bin/clair-scanner
```

Comment:
Instead of doing this add the binary in to the ap-build image and depend on that new tag?

Reply:
done,